.gitattributes

```diff
@@ -0,0 +1,7 @@
+# See https://git-scm.com/docs/gitattributes for more about git attribute files.
+
```

Comment:
- [ ] You are not following Gitflow which is a requirement in your project. You are following GitHub Flow which you are not supposed to follow in this project :thinking: Kindly follow Gitflow :+1: